docs/UserGuide.md

```diff
@@ -239,45 +260,42 @@ Example:
 * `sales BSBM/100 BSBBT/120` Updates the sales of Brown Sugar Boba Milk `BSBM` to 100 and
  Brown Sugar Boba Black Tea `BSBBT` to 120.
 
-#### Listing the number of drinks sold for the day
+#### 4.2 Listing the number of drinks sold for the day
 Shows a list of all types of drinks sold for the current day.
 
 Format: `sales list`
 
```

Comment:
We are missing Feature 4.3 Find the sales history data based on date documentation.

Reply:
Okay, I've added that in. Thanks!